examples/jenkins/pipeline/README.md

```diff
@@ -157,3 +157,29 @@ To run this example:
 14. Start the pipeline again. Go back to step 11 and repeat.
 
 On the first pipeline run, there will be a delay as Jenkins is instantiated for the project. 
+
+## OpenShift Client Plugin Example
+
+The `openshift-client-plugin-pipeline.yaml` build config references a pipeline that showcases the fluent Jenkins pipeline syntax provided by the
+OpenShift Client Plugin.  The DSL provided by this plugin allows for rich interactions with an OpenShift API Server from Jenkins pipelines.
```

Comment:
Need to note that this will only work on the centos jenkins image today since the rhel jenkins image doesn't include the plugin.